host-src/tool/config.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * dc-tool, a tool for use with the dcload serial loader
+ *
+ * Copyright (C) 2001 Andrew Kieschnick <andrewk@napalm-x.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ *
+ */
+
```

Comment:
No reason to put a copyright header (that is assigning copyright to someone who had nothing to do with the file in the first place) on this file.
```suggestion
```